### PR TITLE
get rtd to install needed packages for documentation

### PR DESCRIPTION
**Pull request by qciaran**

### CODE REVIEW

File: .readthedocs.yaml [hunk 1]
```diff
@@ -30,6 +30,6 @@ sphinx:
 # Optional but recommended, declare the Python requirements required
 # to build your documentation
 # See https://docs.readthedocs.io/en/stable/guides/reproducible-builds.html
-# python:
-#    install:
-#    - requirements: docs/requirements.txt
+python:
+  install:
+    - requirements: requirements.txt
```
Comment (qartik) [inline .readthedocs.yaml:35]: Shouldn't this be?
```suggestion
    - requirements: requirements/docs.txt
```

Comment (qartik) [reply .readthedocs.yaml:35]: or perhaps add docs.txt in your primary requirements file.

Comment (qciaran) [reply .readthedocs.yaml:35]: What it had by default is docs/requirements.txt. So yet another requirements file... We could create one, but I am currently just trying to get rtd to work.

Comment (qciaran) [reply .readthedocs.yaml:35]: I guess I wont be lazy, and I'll add docs/requirements.txt. I'll do the copyright sequence too... :D

Comment (qartik) [reply .readthedocs.yaml:35]: actually I was referring to the existing file: https://github.com/PECOS-packages/PECOS/blob/master/requirements/docs.txt

Comment (qciaran) [reply .readthedocs.yaml:35]: That folder/file is in the `master` branch but was removed in the `development` branch. Although could add it back. Or do what sphinx suggested by default (i.e., `docs/requirements.txt`) and copy some of those in there. hmmmm

Comment (qartik) [reply .readthedocs.yaml:35]: Ah, I didn't realize that.

**Comment by qciaran** [comment]
Getting rtd to autobuild docs

**Pull request merged**